sa/authz.go

```diff
@@ -0,0 +1,82 @@
+package sa
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	gorp "github.com/letsencrypt/boulder/Godeps/_workspace/src/gopkg.in/gorp.v1"
+
+	"github.com/letsencrypt/boulder/core"
+)
+
+var authorizationTables = []string{
+	"authz",
+	"pendingAuthorizations",
+}
+
+const getAuthorizationIDsMax = 1000
+
+func getAuthorizationIDsByDomain(db *gorp.DbMap, ident string, now time.Time) ([]string, error) {
```

Comment:
I'm a little worried about intermixing the authorization tables into one array. It seems like a extra work for the DB to do a lot of negative look ups, especially when they aren't de-duplicated.